_stbt/ocr.py

```diff
@@ -347,7 +372,7 @@ def _tesseract_version(output=None):
 
 
 def _tesseract(frame, region, mode, lang, _config, user_patterns, user_words,
-               upsample, text_color, text_color_threshold):
+               upsample, text_color, text_color_threshold, engine):
```

Comment:
Do we want a config option for this too?

Reply:
> Do we want a config option for this too?

Yes, added in a subsequent commit.